modules/raft/src/main/java/org/apache/ignite/raft/jraft/RaftGroupService.java

```diff
@@ -76,26 +76,51 @@ public class RaftGroupService {
      * @param nodeOptions Node options.
      * @param rpcServer RPC server.
      * @param nodeManager Node manager.
-     * @param ownFsmCallerExecutorDisruptorConfig Configuration own striped disruptor for FSMCaller service of raft node, {@code null}
-     *      means use shared disruptor.
      */
     public RaftGroupService(
             final String groupId,
             final PeerId serverId,
             final NodeOptions nodeOptions,
             final RpcServer rpcServer,
-            final NodeManager nodeManager,
-            @Nullable RaftNodeDisruptorConfiguration ownFsmCallerExecutorDisruptorConfig
+            final NodeManager nodeManager
     ) {
         super();
         this.groupId = groupId;
         this.serverId = serverId;
         this.nodeOptions = nodeOptions;
         this.rpcServer = rpcServer;
         this.nodeManager = nodeManager;
-        this.ownFsmCallerExecutorDisruptorConfig = ownFsmCallerExecutorDisruptorConfig;
+        this.ownFsmCallerExecutorDisruptorConfig = null;
     }
 
+        /**
+         * @param groupId Group Id.
+         * @param serverId Server id.
+         * @param nodeOptions Node options.
+         * @param rpcServer RPC server.
+         * @param nodeManager Node manager.
+         * @param ownFsmCallerExecutorDisruptorConfig Configuration own striped disruptor for FSMCaller service of raft node.
+         */
+        public RaftGroupService(
+                final String groupId,
+                final PeerId serverId,
+                final NodeOptions nodeOptions,
+                final RpcServer rpcServer,
+                final NodeManager nodeManager,
+                final RaftNodeDisruptorConfiguration ownFsmCallerExecutorDisruptorConfig
+        ) {
+            super();
```

Comment:
Sorry, I did not express this clearly: the idea was to reuse one constructor inside another. Looks like the only thing that prevents this is the `assert`, but it doesn't seem to be too important. By removing it, we can remove the duplication. WDYT?

Reply:
Fix it.